challenge-find-the-largest-number/script.js

```diff
@@ -1,9 +1,18 @@
 function calculateLargestNumber(arrayOfNumbers) {
-  // Write your code in here
+  let largestNumber= 0;
```

Comment:
You accepted challenge, congrats. You passed for these tests but if an array has all negative number it doesn't work so you need to start 'largestNumber with an element of array'.

Reply:
Thank you. I am very working on it. It is more complicated now.:)